rbs.gemspec

```diff
@@ -34,4 +34,5 @@ Gem::Specification.new do |spec|
   spec.bindir        = "exe"
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
+  spec.required_ruby_version = ">= 2.6"
```

Comment:
I'm concerned that it is inconsistent with `TargetRubyVersion` of .rubocop.yml. It's better to be consistent with minimum version of Ruby (2.6? or 2.7?) that RBS supports.

- https://github.com/ruby/rbs/blob/v1.0.0/.rubocop.yml#L3
- https://github.com/ruby/rbs/pull/184/commits/8fbf6427fce49d6fd8cbcd68731e10ac331ecb30

Reply:
Hmm. What does `TargetRubyVersion` mean for `Rubycw/Rubycw`? @pocke

Reply:
`TargetRubyVersion` does not affect `Rubycw/Rubycw` cop. The cop uses `RubyVM::AST` to detect warnings, so it depends on the runtime Ruby version. If RuboCop runs on Ruby 2.6, the cop checks codes as Ruby 2.6.


But I think it is a good idea to make `TargetRubyVersion` and `required_ruby_version` the same.
It is the best practice of RuboCop and it avoids confusion like this thread.